parsing/tests/parsing.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { beforeEach, describe, expect, test } from '@jest/globals'
+import { processFile } from '@/processor'
+// import { createNewPart } from '@/utils/factory-management/common'
+// import { newFactory } from '@/utils/factory-management/factory'
+// import { addProductToFactory } from '@/utils/factory-management/products'
+
+describe('common', () => {
+//   let mockFactory: Factory
+
+//   beforeEach(() => {
+//     mockFactory = newFactory('Test Factory')
+//     addProductToFactory(mockFactory, {
+//       id: 'CompactedCoal',
+//       amount: 1234,
+//       recipe: 'CompactedCoal',
+//     })
+//   })
+
+  describe('parsing tests', () => {
+    test('parts test', () => {
+        //arrange
+        const inputFile = '../game-docs.json';
+        const outputFile = 'game-docs.json';
+
+        //act
+        let results = processFile(inputFile, outputFile);
+        console.log(results.items.length);
```

Comment:
@Maelstromeous - noob question: I'm trying to understand the jest/Typescript pattern here. How do I expose the `items` in processing to test it?

Reply:
Is it showing as no data or undefined `results`?

Reply:
Items is undefined. I think I need to export it somehow- but I'm not sure how to do that if the class is already exported. In .net id make it a public...

Reply:
The function you're calling doesn't return anything.

Reply:
There's no harm in making it return something, when it's run for real it just writes to the file, it probably should return the contents of said file.

Reply:
Ahhhhhh. I see what you mean. thanks - this is really helpful!!